web/main/migrations/0001_initial.py

```diff
@@ -0,0 +1,97 @@
+# Generated by Django 3.1.1 on 2020-09-14 15:13
```

Comment:
Please remove the migrations.
It should not have been added to the project during development.
Run the makemigrations in the container. 
Best practice is to add to .gitignore https://gist.github.com/santoshpurbey/6f982faf1eacdac153ffd86a3a694239#file-gitignore-L111-L116